src/components/AudioParameter.vue

```diff
@@ -139,7 +139,9 @@ const precisionComputed = computed(() => {
 });
 
 // クリックでアクセント句が選択されないように@click.stopに渡す
-const stopPropagation = undefined;
+const stopPropagation = () => {
+  //
```

Comment:
```suggestion
  // fn is not a function エラーを回避するために何もしない関数を渡す
```
コメント書いておいた方が丁寧かも...?

もしくは改行はたぶん必要ないので、
```
const stopPropagation = () => {};
```
としてしまってもいいかもです

Reply:
コメントなしだとlintがエラーを吐きました…
確かにlintのワーニング消したり謎の空コメント行があるよりはコメントを残す方が分かりやすいと思うので、そちらにしようかと思います！